CONTRIBUTING.md

```diff
@@ -88,8 +87,44 @@ Follow these steps to update a locale for the SAS Extension for VSCode:
 
 ### Locale contributors
 
-| Language                 | VSCode Language ID | Contributor |
-| ------------------------ | ------------------ | ----------- |
-| **German**               | de                 | David Weik  |
-| **Chinese (Simplified)** | zh-cn              | Wei Wu      |
-| **Portuguese (Brazil)**  | pt-br              | Mark Jordan |
+| Language                 | VSCode Language ID | Contributor     |
+| ------------------------ | ------------------ | --------------- |
+| **French**               | fr                 | Vincent Rejany  |
+| **Italian**              | it                 | Lorenzo Roccato |
+| **German**               | de                 | David Weik      |
+| **Chinese (Simplified)** | zh-cn              | Wei Wu          |
+| **Japanese**             | ja                 | Masayuki Iizuka |
+| **Korean**               | ko                 | Meilan Ji       |
+| **Portuguese (Brazil)**  | pt-br              | Mark Jordan     |
+
+## Run single test file
+
+- Open the `.test.ts` file you want to run
+- Switch to the `Run and Debug` view in the VS Code Activity Bar (Ctrl+Shift+D).
+- Select `Language Server E2E Test` from the drop down.
+- Press ▷ to run the launch config (F5).
+- See test result in the `Debug Console` panel.
+
+# Testing unpublished versions
+
+You can still try out any commit or pull request if you don't want to manually build from source code.
```

Comment:
```suggestion
You can still try out any commit or pull request (PR) if you don't want to manually build from source code.
```

Suggesting adding PR here since we use the abbreviation below